tests/ops/op_math/test_composite.py

```diff
@@ -353,15 +353,17 @@ def test_overlapping_ops_property(self):
         valid_op = ValidOp(
             qml.sum(qml.PauliX(0), qml.PauliY(5), qml.PauliZ(10)),
             qml.sum(qml.PauliX(1), qml.PauliY(4), qml.PauliZ(6)),
-            qml.prod(qml.PauliX(10), qml.PauliY(2), qml.PauliZ(7)),
+            qml.prod(qml.PauliX(10), qml.PauliY(2)),
             qml.PauliY(7),
+            qml.prod(qml.PauliX(2), qml.PauliZ(7)),
             qml.prod(qml.PauliX(4), qml.PauliY(3), qml.PauliZ(8)),
         )
```

Comment:
In the old iterative approach, `Y(7)` would've been put in a different group from `X(10) @ Y(2)`, which would've been wrong because `X(2) @ Z(7)` will be then placed in the same group as `X(10) @ Y(2)`, which leads to this group having overlapping wires `Y(7)` in the other group.

Reply:
I know it's the same code but would it make sense to add a `LinearCombination` test case?